src/main/java/plugily/projects/buildbattle/arena/impl/BaseArena.java

```diff
@@ -234,6 +234,9 @@ public void removePlayer(Player player) {
 
   public void clearPlayers() {
     players.clear();
+    for(Player player: players){
```

Comment:
`players` is empty

Reply:
I'm stupid haha. I'll fix it all tomorrow.